dev-utils/poly-cli/src/static/templates/preview/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useEffect } from "react";
+import { createRoot } from "react-dom/client";
+import {
+    Slideshow,
+    Screen,
+    ClickableCard,
+    ProgressBanner,
+    SideSwiper,
+    SideSheet,
+    ProgressIcon,
+} from "@polypoly-eu/poly-look";
+import content from "./static/content.json";
```

Comment:
🤔 I see what you are saying. But I want to keep this free of code, so that designers have an easy time changing things around.

Reply:
Yeah, re reading it, I don't think I understand.

Reply:
JSON is code. Only you're assigning it here instead of there. Just changing it to export const varname = JSON will allow us to avoid the JSON dependency, and will not be more complicated.

Reply:
Now I get it